types/powerapps-component-framework/tslint.json

```diff
@@ -0,0 +1,20 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
No rules allowed in new library packages, please remove these

Reply:
Extra rules removed and fixed the linting errors caused.